app/include/zmk/behavior.h

```diff
@@ -10,4 +10,10 @@ struct zmk_behavior_binding {
     char *behavior_dev;
     u32_t param1;
     u32_t param2;
+};
+
+struct zmk_behavior_binding_data {
```

Comment:
The naming I'm still not 100% happy with.... Thoughts on:

```suggestion
struct zmk_behavior_binding_event_data {
```

Reply:
let's drop the 'data' part?

Reply:
```

struct zmk_behavior_binding_event {
    struct zmk_behavior_binding* binding;
    int layer;
    u32_t position;
    s64_t timestamp;
};
```
this may be an option too...